docker/exasol/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM exasol/docker-db:latest-7.1
+RUN yum update -y --exclude=kernel* && \
+    yum install -y nc
```

Comment:
Any chance we can avoid the build here? Updating dependencies in Docker can make failures hard to debug, so if possible I'd like to use the image directly instead of having a build step where it is isn't necessary.

Reply:
I was able to remove this and use the plain image. This was here to add the `nc` binary for a healthcheck. I altered the health check to run a simple SQL query instead, using `exaplus`.